src/main/bundles/alphabetical.ts

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
would this file not be better kept with the name `back-to-top`? then if we need another back to top we can re-use?

Reply:
I've now moved the BackToTop class to a separate file, which is now called from the alphabetical.ts one (this one still needs to be there as it's an entrypoint for webpack)